contents/blog/slack-closure.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2023-20-12
+title: "Why we're closing our public Slack community"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: CEO diaries
+author: ["joe-martin"]
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+---
+
+One of the great things about PostHog being open source is that we’ve grown a vibrant community around the project. Since launch we’ve accepted code from over 500 contributors and swapped ideas with thousands of users in our public Slack group. 
+
+Now though, we’ve decided to close our public Slack group and focus efforts elsewhere. In this post I’ll explain why we’re doing this, and what comes next for the PostHog community. 
+
+> **Tl;dr:** We’re closing the Slack group on XX/XX/XX and inviting members to join us in the new [PostHog Community](/questions) instead. This decision only impacts the public Slack group, not the private Slack channels for users spending over $2,000 per month.
+
+## Why are we closing the PostHog Slack?
+
+PostHog has grown incredibly fast over the past four years and while that has ensured the public Slack group has remained a busy place, we think it’s arguably become _too_ busy. We often find users asking questions that have already been answered in previous chats, or re-asking unanswered questions if they drift too far back in the chat history. 
```

Comment:
You shared graphs showing that actually the opposite was happening though and messages are generally declining?

Reply:
I don't think these two things are mutually exclusive. The number of messages is declining, but the number of messages is _still_ too much because we don't actively support these users (and the other points mentioned).